benchmarks/torchbench_model.py

```diff
@@ -207,16 +215,9 @@ def load_benchmark(self):
     # workaround "RuntimeError: not allowed to set torch.backends.cudnn flags"
     # torch.backends.__allow_nonbracketed_mutation_flag = True
 
-    if self.benchmark_experiment.accelerator == "cpu":
-      device = "cpu"
-    elif self.benchmark_experiment.accelerator == "cuda" and not self.benchmark_experiment.xla:
-      device = "cuda"
-    else:
-      device = str(self.benchmark_experiment.get_device())
-
     return benchmark_cls(
```

Comment:
Is it possible to force bnechmark_cls to only run once so that we don't have the problem next time? Perhaps https://stackoverflow.com/questions/4103773/efficient-way-of-having-a-function-only-execute-once-in-a-loop

Reply:
Sure. I will do it in a follow-up PR.